integration_tests/test_pv_connection.py

```diff
@@ -36,7 +36,7 @@ def on_market_cycle(self, market_info):
                 assert listed_offer["energy"] == listed_offer["energy"]
                 # Try to delete the offer
                 delete_resp = self.delete_offer(offer_info["id"])
-                assert delete_resp["deleted_offer"] == offer_info["id"]
+                assert delete_resp["deleted_offers"] == [offer_info["id"]]
```

Comment:
Very minor but its better to follow a uniform naming convention, for instance here it is `deleted_offers`  BUT for bids [here](https://github.com/gridsingularity/d3a-api-client/pull/17/files#diff-b154c1ac9b2fffcdf06e03c6c5944d7aR37) it is `bids_deleted`